projects/pt1/python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -4739,5 +4739,37 @@ def forward(self, x):
         return torch.ops.aten.glu(x, dim=1)
 
 @register_test_case(module_factory=lambda: GluStaticModule())
-def  GluStaticModule_basic(module, tu: TestUtils):
+def GluStaticModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 24, 5))
+
+# ==============================================================================
+
+class FakeQuantizePerTensorAffineModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+    @export
+    @annotate_args([
+        None,
+        ([4, 50], torch.float32, True)
```

Comment:
I would expect this decomposition to also work for dynamic shapes. If so, can you add a test with dynamic input?

Reply:
done